cypress.json

```diff
@@ -9,7 +9,7 @@
   "viewportHeight": 1320,
   "env": {
     "openSearchUrl": "http://localhost:9200",
-    "userName": "admin",
-    "password": "admin"
-  }
+    "SECURITY_ENABLED": false
+  },
+  "experimentalNetworkStubbing": true
```

Comment:
This is needed to use the `cy.route2` cmd which is necessary for calls to the AD server.